bitcoin/src/blockdata/script/borrowed.rs

```diff
@@ -189,6 +190,24 @@ impl Script {
             && self.0[24] == OP_CHECKSIG.to_u8()
     }
 
+    /// Checks whether a script is push only.
+    #[inline]
+    pub fn is_push_only(&self) -> bool {
+        for inst in self.instructions() {
+            match inst {
+                Err(_) => return false,
+                Ok(Instruction::Op(op)) => match op.to_u8() {
+                    // From Bitcoin Core:
+                    // if (opcode > OP_16 (0x60)) return false
+                    0..=0x60 => {}
+                    _ => return false,
+                },
```

Comment:
In b5347f8d754d6c2daa287d838af5dcfde1910cad:

I suggest refactoring this so it's not a nested match, like

```
￼            match inst {
￼                Err(_) => return false,
￼                Ok(Instruction::Op(op)) if op.to_u8() <= 0x60 => {}
                // From Bitcoin Core
                // if (opcode > OP_16 (0x60)) return false
                _ => return false,
            },
```

I suspect rustfmt will have some problem with that comment but you can just add `#[rustfmt::skip]`.